client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1774,8 +1775,6 @@
   "report-code.encounters": "Encounters",
   "report-code.expiring-items": "Expiring Items",
   "report-code.inbound": "Inbound",
-  "report-code.expiring-items": "Expiring Items",
-  "report-code.inbound": "Inbound Shipment",
```

Comment:
Was this intentional to remove as part of this PR? Or re added in the base somehow?

Reply:
Yeah sorry should have commented, these are duplicated keys.